Description: change levels location in pacman.h.
Author: Joao Eriberto Mota Filho <eriberto@eriberto.pro.br>
Last-Update: 20130506
diff -Naur pacman4console-1.0.orig/pacman.h pacman4console-1.0/pacman.h
--- pacman4console-1.0.orig/pacman.h	2006-06-12 07:41:40.000000000 -0300
+++ pacman4console-1.0/pacman.h	2006-12-08 11:49:25.000000000 -0200
@@ -1,5 +1,5 @@
 // Some variables that you may want to change
-char LevelFile[50] = "/usr/local/share/pacman/Levels/level__.dat";	//Locations of default levels
+char LevelFile[50] = "/usr/share/pacman4console/Levels/level__.dat";	        //Locations of default levels
 int FreeLife = 1000;							//Starting points for free life
 int Points = 0;								//Initial points
 int Lives = 3;								//Number of lives you start with
